bigtable/snippets/garbagecollection/create_family_gc_intersect.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package garbagecollection
+
+// [START bigtable_create_family_gc_intersection]
+import (
+	"cloud.google.com/go/bigtable"
+	"context"
+	"fmt"
+	"io"
+	"time"
+)
+
+func createFamilyGCIntersect(w io.Writer, projectID, instanceID string, tableName string) error {
```

Comment:
Normally have function comments, but if it's repetitive with the docs surrounding the snippet, they're OK to leave off. https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#comment-functions-and-packages